emboss-format

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Benjamin Olmstead
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Standalone driver program for the Emboss formatter."""
+
+import argparse
+import os
+import sys
+
+
+def main(argv):
+    base_path = os.path.dirname(__file__) or "."
+    sys.path.append(base_path)
+
+    from compiler.front_end import format  # pylint:disable=import-outside-toplevel
```

Comment:
Why import inside main? I know we do this in `embossc` as well, but I wasn't sure the purpose of this.

Reply:
Updated.  Here, I was just copying `embossc`.

When I wrote `embossc`, I thought it was necessary to add the script's directory to Python's module path before doing the import, but it looks like [a standard Python installation will put the script's directory into the module search path](https://docs.python.org/3/library/sys_path_init.html).  Either I was testing `embossc` on a weird installation or I got confused somewhere -- I'd guess the latter 🙂.  Either way, I can send a followup PR to fix that script.